.devcontainer/devcontainer.json

```diff
@@ -28,7 +28,9 @@
 		"terminal.integrated.defaultProfile.linux": "bash",
 		"python.pythonPath": "/usr/local/bin/python",
 		"python.linting.enabled": true,
-		"python.linting.pylintEnabled": true,
+		"python.linting.pylintEnabled": false,
+		"python.linting.flake8Enabled": true,
```

Comment:
I'm all cool with the settings being here (probably better) - but can we remove the settings from .vscode/settings.json - otherwise it will confuse someone (probably me!)

Reply:
Good shout - I'll move/merge them to devcontainer.json

Reply:
done